packages/terra-functional-testing/src/reporters/spec-reporter/wdio-spec-reporter.js

```diff
@@ -120,7 +120,12 @@ class SpecReporter extends WDIOReporter {
     if (spec.indexOf(`${path.sep}packages${path.sep}`) > -1) {
       return spec.split(`${path.sep}packages${path.sep}`).pop().split(path.sep)[0];
     }
-
+    const filepath = `${process.cwd()}${path.sep}package.json`;
```

Comment:
should we use join instead? it automatically normalizes the path
```suggestion
    const packageJsonPath = path.join(process.cwd(), 'package.json');
```